Header header
uint64     PartNumber
uint64     AssemblyPartNumber
uint64     SWPartNumber
uint8[26]  SerialNumber
uint32     BLVersion
uint32     SWVersion
uint64     UTCTimeStamp
uint32     TimeStamp
float32    SurfaceDamping
uint8      OpState
uint8      CurrentFarCF
uint8      CurrentNearCF
uint8      Defective
uint8      BadSupplyVolt
uint8      BadTemp
uint8      GmMissing
uint8      TxPowerStatus
float32    MaximumRangeFar
float32    MaximumRangeNear
